API/API.mli

```diff
@@ -329,6 +343,7 @@ sig
   end
 
   type universe_level = Level.t
+  [@@ocaml.deprecated "Deprecated form, see univ.ml"]
```

Comment:
Can we suggest the replacement in the warning message? (and similarly for others below)

Reply:
Not worth doing more cut and paste, the deprecation notice in `Univ` has the correct message, and the info will be correctly propagated once #6067 is merged.

In general, the `API.mli` is yet not release-worthy, I think. Notice that we have to disable warnings etc...

Reply:
Ah ok, sorry, I missed that it would again be duplicated information.

Reply:
Well if we were to release now we should indeed be sure that API.mli is correctly reflecting the changes in the core.

Right now I did not bother as it would take a large amount of time and we don't know yet if we will finally use some form of module `include` to avoid the duplication, making thus this tedious work unnecessary.